codecov.yml

```diff
@@ -0,0 +1,51 @@
+# Setting coverage targets per flag
+coverage:
+  status:
+    project:
+      default:
+        target: 80%  # overall project/ repo coverage
+
+      rest-gateway:
+        target: auto
+        flags:
+          - rest-gateway
+
+      catbuffer-parser:
+        target: auto
+        flags:
+          - catbuffer-parser
+
+      sdk-python:
+        target: auto
+        flags:
+          - sdk-python
+
+      sdk-javascript:
+        target: auto
+        flags:
+          - sdk-javascript
+
+# New root YAML section = `flags:`
+# This is where you would define every flag from your
+# uploader, and update when new Flags added
+
+flags:
```

Comment:
can you explain what this section does (different from previous)?

Reply:
I can add some more comments to it.  The flags section actually define the flags(root folder for the package) that are used in the projects above.